client/model/pillarModel.js

```diff
@@ -160,6 +161,15 @@ define( [
                 return true;
             });
 
+            // basically verbatim from https://coderwall.com/p/imkqoq/resize-textarea-to-fit-content
+            $(document).on( 'change click keyup keydown paste cut', '.resize', function (){
+                $(this).height(0).height(this.scrollHeight);
+            }).find( '.resize' ).change();
```

Comment:
no, what is your $0.02 on it? anything with resize in the class I want to resize, unless there is a smart way to only change the ones that are about to be displayed (which i tried to find)